components/motor/gpio/loop.go

```diff
@@ -38,149 +23,80 @@ func (m *EncodedMotor) State(ctx context.Context) ([]float64, error) {
 func (m *EncodedMotor) updateControlBlock(ctx context.Context, setPoint, maxVel float64) error {
 	// Update the Trapezoidal Velocity Profile block with the given maxVel for velocity control
 	velConf := control.BlockConfig{
-		Name: m.blockNames[blockNameTrapezoidal],
+		Name: m.blockNames["trapezoidalVelocityProfile"][0],
```

Comment:
Why take out the named constants? In the old version, if you had a typo on this line, the code wouldn't compile. In the new version it would compile but panic when run, which is harder to track down.

Reply:
it was feeling a little redundant because I know it will always be this exact string. but I can readd it to avoid future potential issues

Reply:
Two jobs ago, my coworkers on the motion team spent months working on a bug that could have been prevented by this kind of change (the movement was off by a small but important amount at the end, and they tried changing their algorithms, and updating their movement based on what the cameras were seeing, and a bunch of other stuff, and the real problem was just that they were using `m.blockNames["trapezoidalVelosityProfile"]` which defaulted to 0 instead of the value that had been set elsewhere). Enums were invented in part to fix this class of bug, which makes me think it's not just my old coworkers who struggled with it.

Reply:
I changed it